e2e/storybook-angular/src/storybook-angular.test.ts

```diff
@@ -2,321 +2,120 @@ import {
   checkFilesExist,
   cleanupProject,
   getPackageManagerCommand,
+  getSelectedPackageManager,
   isNotWindows,
   killPorts,
-  newProject,
-  readFile,
   runCLI,
   runCommand,
+  runCreateWorkspace,
   runCypressTests,
   tmpProjPath,
   uniq,
-  updateJson,
 } from '@nrwl/e2e/utils';
 import { writeFileSync } from 'fs';
 
 describe('Storybook for Angular', () => {
-  let proj: string;
+  const previousPM = process.env.SELECTED_PM;
+  const packageManager = getSelectedPackageManager() || 'yarn';
+  const proj = uniq('proj');
+  const appName = uniq('app');
 
   beforeAll(() => {
-    proj = newProject();
-
-    // TODO(jack): Overriding enhanced-resolve to 5.10.0 now until the package is fixed.
-    // See: https://github.com/webpack/enhanced-resolve/issues/362
-    updateJson('package.json', (json) => {
-      if (process.env.SELECTED_PM === 'yarn') {
-        json['resolutions'] = {
-          'enhanced-resolve': '5.10.0',
-        };
-      } else if (process.env.SELECTED_PM === 'npm') {
-        json['overrides'] = {
-          'enhanced-resolve': '5.10.0',
-        };
-      } else {
-        json['pnpm'] = {
-          overrides: {
-            'enhanced-resolve': '5.10.0',
-          },
-        };
-      }
-      return json;
+    process.env.SELECTED_PM = 'yarn';
```

Comment:
We need to test if this functionality works across all the package managers. This way we are constantly checking just `yarn`.

Reply:
`pnpm` is not supported by 6.5, it needs Storybook 7

Reply:
https://github.com/storybookjs/storybook/issues/13428#issuecomment-1368062647